custom_components/adjustable_bed/brand_registry.py

```
@@ -0,0 +1,404 @@
+
+    Returns:
+        The protocol file name (without .py extension)
+
+    Raises:
+        ValueError: If bed_type is unknown
+    """
+    # Check legacy aliases first
+    canonical_type = LEGACY_ALIASES.get(bed_type, bed_type)
+
+    if canonical_type not in BRANDS:
+        raise ValueError(f"Unknown bed type: {bed_type}")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)